### ISSUE TITLE
Pull request for TER release

### ISSUE THREAD

**Issue by christophlehmann** [AgenceStratis/TYPO3.be_acl#3]
What about a pull request for a TER release?

**Comment by fedir** [comment]
Yep, this could be done, as soon the extension will be well tested and fixed.

**Comment by fedir** [comment]
@nicozerr Please confirm, if the extension is fully operational, or we need to adjust it still.

**Comment by nicozerr** [comment]
For me, it's all good.
